tsdb/compact.go

```diff
@@ -84,28 +84,28 @@ type LeveledCompactor struct {
 	mergeFunc                storage.VerticalChunkSeriesMergeFunc
 }
 
-type compactorMetrics struct {
+type CompactorMetrics struct {
 	ran               prometheus.Counter
-	populatingBlocks  prometheus.Gauge
-	overlappingBlocks prometheus.Counter
+	PopulatingBlocks  prometheus.Gauge
+	OverlappingBlocks prometheus.Counter
 	duration          prometheus.Histogram
 	chunkSize         prometheus.Histogram
 	chunkSamples      prometheus.Histogram
 	chunkRange        prometheus.Histogram
```

Comment:
It makes sense to expose all the metrics at this point instead of a selected few